benchmarks/src/twitter_data.hpp

```diff
@@ -14,30 +14,14 @@ struct User {
     int64_t followers_count;
     int64_t friends_count;
     int64_t statuses_count;
-
-    bool operator==(const User &other) const {
-        return id == other.id &&
-               name == other.name &&
-               screen_name == other.screen_name &&
-               location == other.location &&
-               description == other.description &&
-               verified == other.verified &&
-               followers_count == other.followers_count &&
-               friends_count == other.friends_count &&
-               statuses_count == other.statuses_count;
-    }
+    bool operator <=>(const User &other) const = default;
```

Comment:
Changes look good to me, helped me learn a bit more about the spaceship operator. Code also got much smaller/cleaner.

 Can you elaborate on why using the spaceship operator for some of the structs and the "==" operator for others?

Reply:
> Can you elaborate on why using the spaceship operator for some of the structs and the "==" operator for others?

I could have used `operator==` throughout but `operator<=>` is more general as it includes other comparisons. That is the 'modern C++' way.  I tried using it throughout but got some compile-time errors where `std::vector` was involved. I don't understand why: `operator<=>` is defined for `std::vector`. So I fell back on `operator==`. I don't have a good answer. It just did not work for some reason. (Maybe I did something wrong.)